aqua/constants.aqua

```diff
@@ -0,0 +1,3 @@
+const REPLICATION_FACTOR = 1
+const CONSISTENCY_LEVEL = 1
```

Comment:
I wish it would be possible to say something like "QUORUM" so it would take `n/2 + 1` of the neighborhood.

Reply:
I don't how to express this in constant but we can have dynamic consistency level based on the size of current resource_id neighborhood

Reply:
you can. some distributed dbs do it like that, e.g., https://docs.scylladb.com/stable/cql/consistency.html (and checkout the calculator)

Reply:
Okay, I will create an issue and do it in another PR